api/src/repositories/admin/admin.repository.ts

```diff
@@ -0,0 +1,166 @@
+import {Service} from 'typedi';
+
+import {Db} from '../../db/db.class';
+import {knex} from '../../db';
+
+@Service()
+export class AdminRepository {
+  constructor(
+   private db: Db,
+  ) {}
+  
+  /**
+   * This is used to bind the user inputted value to prevent
+   * SQL injection attempts while using a Knex Raw
+   * @param field
+   * @param value
+   * @private
+   */
+  private like(field: string, value: string) {
```

Comment:
Private methods always go beneath public methods.

Reply:
Noted and will fix